CI/templates/_job_template.yaml

```diff
@@ -9,6 +9,10 @@ parameters:
   job_display_name: ""
   operating_system: "" # Windows|Linux|MacOS
   steps: []
+  submodule_checkout: 

```

Comment:
There are pipelines that use this template (see the Publish pipelines) that won't want to checkout the code. Those yaml files should be updated to override this parameter.